cldfbench_doreco.py

```diff
@@ -171,15 +172,15 @@ def create_schema(self, cldf):
             'LanguageTable',
             {
                 'name': 'Source',
-                'datatype': 'str',
+                'datatype': 'string',
```

Comment:
@Tarotis the CSVW datatype is called [string](https://csvw.readthedocs.io/en/latest/datatypes.html#csvw.datatypes.string).